README.md

```diff
@@ -46,6 +46,39 @@ if everything is working.
 
 ![screenshot](https://raw.githubusercontent.com/thaJeztah/pgadmin4-docker/master/pgadmin-screenshot.png)
 
+### Connect to a host running PostgreSQL
+
+To connect pgadmin4 with a database running in a host you have to modify two files in the host, pg-hba.conf and postgresql.conf
```

Comment:
Can you;

- change "pgadmin4" to "pgAdmin 4" (capital "A", and a space before the "4"; it's the preferred naming that's used)
- put the `pg-hba.conf` and `postgresql.conf` in back tics (here, and in the other lines below) so that they are presented in a fixed-width font? i.e.;

```Markdown
To connect pgadmin4 with a database running in a host you have to modify two files in the host, `pg-hba.conf` and `postgresql.conf`
```

- wrap long lines to 80-characters?